CONTRIBUTING.rst

```diff
@@ -20,9 +20,55 @@
 Contributions
 =============
 
-Contributions are welcome and are greatly appreciated! Every little bit helps,
+Contributions to Airflow are welcome and are greatly appreciated! Every little bit helps,
 and credit will always be given.
 
+We would like more people to use, contribute to, and help maintain Airflow.
+These three goals align with the three main membership roles within our community:
+users, contributors, and committers/maintainers.
+
+We would like to instill a sense of ownership and pride among the members of our community
+by encouraging:
+
+- people to use Airflow
+- users to contribute ro Airflow
+- contributors to become committers
+
+You can contribute to Airflow in various ways such as:
+
+- reporting bug
+- suggesting new features and Road Map items
+- submitting Pull Requests (PRs) for bug fixes and new features.
+- attending Airflow meets-up events near you:
+
+  - search for Airflow `Meetup <https://www.meetup.com/>`_
+
+- helping to answer questions on gitter, the dev email list, PRs, and Jiras.
```

Comment:
Gitter and Jira are not used any more.

Reply:
Indeed. We have Slack, devlist, Discussions in GitHub, Gitihub Issues and PRs.